pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
-    "pack_format": 2,
+    "pack_format": 3,
```

Comment:
Pack format 3 enforces lower-casing, and CTP has non-lower-case files. This change will break `soulshardstow/blockstates/BlockCage` and `enderio/blockstates/blockIngotStorage`